AnimateSvg.js

```diff
@@ -46,56 +96,18 @@ export default function AnimateSvg(props) {
         useNativeDriver: true,
       }),
     ]).start();
-    const endOfAnim = setTimeout(() => {
```

Comment:
setTimeout removed in favor of Animation start() method callback in order to perform same logic.

Reply:
🌮 🌮 🌮 🌮